common/views/components/SearchForm/SearchForm.tsx

```diff
@@ -291,7 +290,7 @@ const SearchForm = forwardRef(
           )}
         </noscript>
         <SearchButtonWrapper>
-          <ButtonSolid text="Search" size="large" />
+          <ButtonSolid text="Search" size="medium" />
```

Comment:
Again, was using the "old" large.

Before
<img width="1248" alt="searchtabs-before" src="https://user-images.githubusercontent.com/110461050/201322360-91d39865-c563-462f-8262-80fcdc99a144.png">

After
<img width="1062" alt="Screenshot 2022-11-11 at 10 35 00" src="https://user-images.githubusercontent.com/110461050/201322488-21a792cc-9369-4913-bf15-8a04727d59ed.png">